apps/src/aichat/views/model-customization-workspace.module.scss

```diff
@@ -11,6 +11,15 @@
   display: flex;
   flex-direction: column;
 
+  .tabsContainer {
+    background-color: #EAEBEB
```

Comment:
This hard-coded color will be added by @levadadenys soon and can update when it is.